transport/interface.go

```diff
@@ -40,6 +40,11 @@ type Request struct {
 	Body             []byte
 }
 
+// StreamRequest is a wrapper of Request, to be used for streaming RPC
+type StreamRequest struct {
+	Request *Request
+}
+
```

Comment:
We can even do better with `type StreamRequest Request`

Reply:
sync offline.

We decided to keep it like this to have a more flexible way of enhancing `StreamRequest` in the future if needed